Source/common/SNTRule.h

```diff
@@ -49,4 +55,9 @@
                           type:(SNTRuleType)type
                      customMsg:(NSString *)customMsg;
 
+///
+///  Sets timestamp of rule to the current time.
+///
+- (void)refreshTimestamp;
```

Comment:
Renamed `refreshTimestamp` to `resetTimestamp`.  I made timestamp into a readonly property, but SNTRuleTable needs to be able to set it with results from the rules database.  So this meant adding an extra initializer for SNTRule.

I like having the method to reset the timestamp inside of SNTRule so that the logic for setting the timestamp value is self-contained.  Otherwise we'd have to duplicate the code to set the timestamp in two places, which could lead to errors if we ever decided to change the format.